$NetBSD: patch-ac,v 1.1 2001/01/01 14:30:25 itojun Exp $

Everything pkgsrc builds on counts as "unix", but doesn't necessarily
pollute the namespace with that by default.

--- lirec/util.h.orig	2000-06-12 22:20:14.000000000 +0000
+++ lirec/util.h
@@ -53,7 +53,7 @@ char	*tempstring();	/* returns a pointer
 #	define GRAPHICS		/* only GDEV on unix machines */
 #endif
 
-#ifndef unix
+#if 0
 
 /* various BSD to lattice C name changes */
 
